testsuite/cluster-test/src/experiments/performance_benchmark.rs

```diff
@@ -61,6 +61,8 @@ pub struct PerformanceBenchmarkParams {
     pub backup: bool,
     #[structopt(long, default_value = "0", help = "Set gas price in tx")]
     pub gas_price: u64,
+    #[structopt(long, help = "Set periodic stat aggregator step")]
+    pub periodic_step: Option<u64>,
```

Comment:
nit - can you rename it to `periodic_stats`? `periodic_step` does not set context for ppl who did not see this PR